src/platforms/evdev/libinput_device.cpp

```diff
@@ -694,3 +694,16 @@ void mie::LibInputDevice::apply_settings(mi::TouchscreenSettings const& settings
 {
     touchscreen = settings;
 }
+
+void mie::LibInputDevice::set_leds(mir::input::KeyboardLeds leds)
+{
+    int led = 0;
```

Comment:
Can't we avoid the `static_cast` later with:

```suggestion
    libinput_led led = 0;
```

Reply:
The compiler gets mad, saying:

```
Cannot initialize a variable of type 'libinput_led' with an rvalue of type 'int'
```

And then all of the `|=` don't work too:

```
Assigning to 'libinput_led' from incompatible type 'int'
```